src/recommendations/test/integ/test_discounted.py

```diff
@@ -7,13 +7,16 @@
     post_request_assert,
     read_file
 )
+from dotenv import load_dotenv
+
+load_dotenv()
+
+DEFAULT_LOCAL_API = 'http://localhost:8005'
 
 cwd = os.path.dirname(os.path.abspath(__file__))
 request_bodies_path = absolute_file_path(cwd, "json_request_bodies.json")
 schemas_path = absolute_file_path(cwd, "json_schemas.json")
-recommendation_api_url = os.getenv("RECOMMENDATIONS_API_URL") or sys.exit(
-    "Please provide an environment variable RECOMMENDATIONS_API_URL"
-)
+recommendations_api_url = os.getenv("RECOMMENDATIONS_API_URL", DEFAULT_LOCAL_API)
```

Comment:
All the changes in `recommendations` is just to use `dotenv` and make it consistent with the other three.